pykickstart/sections.py

```diff
@@ -676,7 +676,7 @@ def _getParser(self):
         if self.version < F24:
             return op
 
-        op.add_argument("--excludeWeakdeps", dest="excludeWeakdeps",
+        op.add_argument("--excludeWeakdeps", "--exclude-weakdeps", dest="excludeWeakdeps",
```

Comment:
Could you add tests for these two options, please?

Reply:
I agree. And it looks like the existing tests may need a little work, I don't see it testing --excludeWeakdeps on the input side, only that it is added by setting pkgs.excludeWeakdeps

If we are going to make this change we should also change the output so that it uses the new form. And while I'm thinking of things to do :) maybe it would be best to add the alias, and deprecate the old version so we can remove it later? Ideally this would all be done in the F32 part of PackageSection instead of where it was originally defined. Looking at the existing code I'm not sure we've done something like that before.